filtering/check_glpi_computers.py

```diff
@@ -71,11 +71,8 @@ def main() -> None:
 
     urls = UrlInitialization(ip)
 
-    session_object = SessionHandler(user_token, urls.INIT_URL, urls.KILL_URL, no_verify)
-    session = session_object.session
-    print(get_computers(session, urls))
-
-    del session_object
+    with SessionHandler(user_token, urls.INIT_URL, urls.KILL_URL, no_verify) as session:
```

Comment:
You could probably just pass urls into the init to make it a little neater but thats not really that important.

Reply:
Yup, agree, done in the latest commit